### PR TITLE
Move to packaging version

### PR DESCRIPTION
**Pull request by typemytype**

### CODE REVIEW

File: setupApp.py [hunk 1]
```diff
@@ -56,27 +55,6 @@ def _findModules(root, extensions, skip, parent=""):
                 else:
                     yield moduleName
 
-def getStdLibModules():
```
Comment (justvanrossum) [inline setupApp.py:59]: Why this change?

Comment (typemytype) [reply setupApp.py:59]: oh will revert... this is result of research done with the latest py2app and embedding std module. It seems not be necessary to explicitly list every module (will be a different PR)
